docs/dev/developer-guides/security.md

```diff
@@ -0,0 +1,38 @@
+# Security model
+
+Given that zkSync is still in its early stages of development, there is a chance that our codebase contains undiscovered bugs that could lead to the loss of users' assets. As a result, we are constantly working to strengthen our security protocols to ensure that users and developers have a seamless onboarding experience when building on zkSync.
+zkSync smart contract codebase is presently being audited, but audits are not a guarantee of quality, and they are not a guarantee that a codebase has been thoroughly vetted for vulnerabilities. It's crucial to realize that zkSync usage exposes you to the possibility of defects in the zkSync codebase.
+
+## Bug Bounty Program
+zkSync takes security seriously and as such, we have a massive [bug bounty program](https://immunefi.com/bounty/zksync/). 
+
+## Scope
+Issues that can lead to substantial loss of money, critical bugs like a broken live-ness condition, blocking upgrade system, irreversible loss of funds, or enforced exodus mode.
+
+## Assumptions
+
+To be eligible for the bug bounty, a bug should adhere to the security assumptions of zkSync. You can read more about them [here](https://github.com/matter-labs/zksync/blob/master/docs/protocol.md#assumptions)
```

Comment:
Does linking to a v2 doc makes sense?

Reply:
It's not bad, actually.

Reply:
ops... wrong comment.. I meant does linking to a v1 doc makes sense? Is it really valid?